ESP32-mqtt-room.ino

```diff
@@ -305,10 +305,11 @@ class MyAdvertisedDeviceCallbacks: public BLEAdvertisedDeviceCallbacks {
 	void onResult(BLEAdvertisedDevice advertisedDevice) {
 
 		digitalWrite(LED_BUILTIN, LED_ON);
-		// Serial.printf("Advertised Device: %s \n", advertisedDevice.toString().c_str());
-		vTaskDelay(10 / portTICK_PERIOD_MS);
+		Serial.printf("Advertised Device: %s \n", advertisedDevice.toString().c_str());
+		if (mqttClient.connected()) {
+			reportDevice(advertisedDevice);
```

Comment:
Reporting device over MQTT during BLE scan will likely cause crashes. There is an issue with coexistence as both WiFi and Bluetooth use the same antenna. All devices are only reported after scan has ended for this reason.

Reply:
I never got any crashes in my testing. Based on this http://bbs.esp32.com/viewtopic.php?f=13&t=6707&sid=30ede9a665f471d433dec8df587c24f0&start=30#p29419 I thought it should by ok if I keep scan window smaller that scan interval.

Reply:
Would this patch be acceptable if I introduced a parameter in settings to select when devices will be reported?

Reply:
I have discounted the idea of reporting during the scan, as it caused no end of issues with stability. I can pull your branch and test locally. It's an insidious issue that only appears intermittently, and may be related to your local environment. For example, I'm in an apartment building where there are loads of wireless networks and bluetooth devices all comingled.

Reply:
OK, I suppose you tested this more that me so I removed this change, leaving only modification of ibeacon id and scanner parameters in Setting.h. Is this acceptable? If yes - should I close this pull request, squash unnecessary commits and create new PR?

Reply:
Sorry for the delay in response - life has gotten in the way more than I had anticipated.

A new PR would certainly help me get these changes merged into the repo.